sdk/cognitiveservices/cognitiveservices-textanalytics/README.md

```diff
@@ -2,11 +2,15 @@
 
 This package contains an isomorphic SDK for TextAnalyticsClient.
 
+> Please note, a newer package [@azure/ai-text-analytics](https://www.npmjs.com/package/@azure/ai-text-analytics) is available as of June, 2020. While this package will continue to receive critical bug fixes, we strongly encourage you to upgrade.
+
```

Comment:
We can mention the API versions as follows:
```suggestion
> Please note, a newer package [@azure/ai-text-analytics](https://www.npmjs.com/package/@azure/ai-text-analytics) is available as of June, 2020 and uses the newer Text Analytics API v3.0 and up. While this package will continue to receive critical bug fixes, it uses the older API v2.1 and we strongly encourage you to upgrade.

```

Reply:
Nits:

"uses Text Analytics API v3.0 or newer." (slightly shorter and clearer)

"v2.1, and we strongly encourage" (comma)

Reply:
Fixed in https://github.com/Azure/azure-sdk-for-js/pull/20262/commits/7d669615082cc9225dcb0c90ee3eb50b90798b41